### DISCUSSION TITLE
PeakAnnoEnrichment dataset

### DISCUSSION THREAD

**Discussion by NoemieL** [GreenleafLab/ArchR#1075] [Questions / Documentation]
Hi, I am interested to know which peaks from my dataset are also on the dataset used for the PeakAnnoEnrichment  analysis. For exemple I would like to know the peaks that are found in B_bulk from Encode ATACbulk. Do you know where I could find this information? Thanks

MM_OnlyPre_peak <- addArchRAnnotations(ArchRProj = MM_OnlyPre_peak, collection = "ATAC",force=TRUE)

enrichATAC <- peakAnnoEnrichment(
  seMarker = markersPeaks,
  ArchRProj = MM_OnlyPre_peak,
  peakAnnotation = "ATAC",
  cutOff = "FDR <= 0.05 & Log2FC >= 1"
)

[ATAC-Enriched-Marker-Heatmap_Translocation_Log2FC>=1_n10_rnareintegration.pdf](https://github.com/GreenleafLab/ArchR/files/7340578/ATAC-Enriched-Marker-Heatmap_Translocation_Log2FC.1_n10_rnareintegration.pdf)

**Comment by rcorces** [comment]
I'm not sure I fully understand the question. But why not just use standard GRanges operations to look at which peaks overlap?
